<!-- path recorder launch -->
<launch >
	<!-- path recorder -->
	<node name="path_recorder" pkg="path_follower" type="path_recorder" output="screen" >
		<param name="/map_frame" value="/world" type="string" />
		<param name="/base_link_frame" value="/base_link" type="string" />
		<param name="/output_path_topic_name" value="/recorded_path" type="string" />
		<param name="/dist_interval" value="0.5" type="double" />
		<param name="/angle_interval" value="10.0" type="double" />
	</node >
</launch >
